ProductivitySuite/tests/test_productivity_suite_on_xeon.sh

```diff
@@ -0,0 +1,403 @@
+#!/bin/bash
```

Comment:
The test scripts name should default to be `test_productivitysuite_on_xeon.sh`, so this script will be auto triggered.

Reply:
test_${example_name_low_letter}_on_xeon.sh